Blog post: Static sites

machikoyasuda opened the pull request:
closes #249 

Blog post writer: Machiko Yasuda
Text: In this PR

Headline: How Compiler approaches static sites
Subhead: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

Social media blurb: Learn how the Engineering team combines older and reliable static site frameworks with new build tools and an open source approach to craft memorable and functional content-driven sites.

machikoyasuda commented:
This post is ready for editing. @laneymangan

angela-tran commented on src/_posts/2024-11-01-static-sites-at-compiler.md:
```diff
@@ -0,0 +1,62 @@
+Writing a static site with Jekyll means that developers ultimately have to initiate and deploy the site, unlike a tool like WordPress or Webflow where the people writing the actual content can be in charge of publishing. With our stack, however, we use GitHub and Netlify to get designers, copy writers and stakeholders involved in more parts of the site creating process:
+
+- Anyone can create a GitHub ticket to request new content to be added, content to be edited.
+- GitHub issues can be customized to ensure developres are getting all the data, metadata they need from stakeholders to make the needed updates. Here's an example of a [issue template for a recent Cal-ITP project](https://github.com/cal-itp/calitp.org/issues/new?template=new_resource.yml).
```
```suggestion
- GitHub issues can be customized to ensure developers are getting all the data, metadata they need from stakeholders to make the needed updates. Here's an example of a [issue template for a recent Cal-ITP project](https://github.com/cal-itp/calitp.org/issues/new?template=new_resource.yml).
```

machikoyasuda commented:
@laneymangan @angela-tran I updated the copy with the latest from Google Docs. Thank you for all the edits.

I changed the date to December 3rd.... which, sorry, I realized, makes the preview link not work.

The best way to preview the code/text now is to look here: https://github.com/compilerla/compiler.la/blob/19f4959fb153e01fb0a096d54ef9b62c12348b0c/src/_posts/2024-11-01-static-sites-at-compiler.md

angela-tran commented:
I just manually triggered a new deploy preview in Netlify for this PR so that we can preview the post. :eyes:
